### ISSUE TITLE
passive rushee doesnt talk to cain at a2 

### ISSUE THREAD

**Issue by Sherloock** [kolton/d2bot-with-kolbot#720]
So i have 1rusher and 4rushee (one of them doing the quests Config.Rushee.Quester = true) the other 3 is passive (Config.Rushee.Quester = false), they cant bypass a2, beacuse they cant go to the palace portal. If i speak with Cain manually everything works fine.

**Comment by DarkHorseDre** [comment]
It's been a long time since I rushed but I had same issue. Not sure of the cause but I had to set them all to quest.
I have seen fixes to this area recently so ensure you are at latest trunk release.

**Comment by DarkHorseDre** [comment]
Guys, is this only an issue with non questing rushees? (therefore, setting all to questing avoids the issue?)

**Comment by krazy03** [comment]
yes only with non questing rushees, the problem is if you have more than 3 rushee with quest set to true they get in each other ways, and what was happening was that some wouldn't get the items and the others would only get 1 out of items needed to make staff... so if you had 5 rushees no one could get full staff because they all had only 1 part or none not sure why it happens. I'm guessing script doesn't check to make sure quest item is gotten or it fails to get it  but no error appears.

**Comment by DarkHorseDre** [comment]
Ah ok - thats why I didnt fail as I only had 1/2 questing for socket :)

Yeah there isn't any bi-directional communication between rusher and rushee and the rusher doesn't know which rushee has which component for quest.  The staff quest is the tricky one (so would act 3 organs but rusher can bypass).

Its good you guys found a workaround as coding to handle this properly would be a bit of work I'd imagine!
